sys/include/net/gnrc/netif/hdr.h

```diff
@@ -199,6 +199,32 @@ gnrc_pktsnip_t *gnrc_netif_hdr_build(uint8_t *src, uint8_t src_len, uint8_t *dst
  */
 void gnrc_netif_hdr_print(gnrc_netif_hdr_t *hdr);
 
+/* @brief   Fetch the netif header flags of a gnrc packet
+ *
+ * @param[in]   pkt     gnrc packet from whom to fetch
+ *
+ * @return              the netif header flags of @p pkt
+ */
+uint8_t* gnrc_netif_hdr_get_flag(gnrc_pktsnip_t* pkt);
```

Comment:
@miri64 For getting the flags, I also returned to the original state of adding the search functionality (the input is`gnrc_pktsnip_t* pkt` ), since I think if the input is already the `gnrc_netif_hdr_t *hdr`, a wrapper of simply returning 'hdr->flags' is unnecessary.  What do you think? ;-)